CRM/Contribute/DAO/Contribution.php

```diff
@@ -322,8 +322,12 @@ static function &fields() {
         'payment_instrument_id' => array(
           'name' => 'payment_instrument_id',
           'type' => CRM_Utils_Type::T_INT,
-          'title' => ts('Payment Method') ,
+          'title' => ts('Payment Method ID') ,
           'description' => 'FK to Payment Instrument',
+          'export' => true,
```

Comment:
honestly - the way the code works we need to really really NOT want to expose stuff before we don't say export = true. api_key is a good example of when we should use false...